specifications/api/swagger.yaml

```diff
@@ -303,7 +303,13 @@ x-settings: &settings
   properties:
     pool_metadata_source:
       <<: *poolMetadataSource
-      description: How to fetch pool metadata
+      description: |
+        Select stake pool metadata fetching strategy:
```

Comment:
The values are already explained here, no?

```yaml
x-poolMetadataSource: &poolMetadataSource
  description: |
    Pool metadata source. This sets the metadata fetching strategy.

    Possible values are
      * null -> no fetching
      * direct -> direct fetching
      * uri -> use SMASH server
  oneOf:
    - type: string
      enum:
        - none
        - direct
    - title: URI
      type: string
      format: uri
      example: https://smash.cardano-mainnet.iohk.io/
```

Reply:
Yes, but I figured it renders nicely in the API doc. :) (see screenshot in Comment in the pr)

Reply:
Hmm, how do we make sure the doc doesn't go async with the other place?

Reply:
There's this -> https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs#L411-L414.

Descriptions are manual maintenance, but having stuff clearly explained improves user experience.